openmc/mgxs/mgxs.py

```diff
@@ -1523,7 +1523,8 @@ def get_pandas_dataframe(self, groups='all', nuclides='all',
 
             df.rename(columns={'energyout low [MeV]': 'group out'},
                       inplace=True)
-            out_groups = np.tile(all_groups, int(df.shape[0] / all_groups.size))
+            out_groups = np.repeat(all_groups, self.xs_tally.num_scores)
+            out_groups = np.tile(out_groups, int(df.shape[0] / out_groups.size))
```

Comment:
Thanks for making this fix! The repeating/tiling can be confusing to wrap one's head around. Could you fix this in `mdgxs.py` on line 808 as well?